### ISSUE TITLE
Error: for input string: "xl"

### ISSUE THREAD

**Issue by Rossella-Mascia-Neosyn** [gluestack/gluestack-ui#2193]
### Description

after updating to the latest version there is an error

### CodeSandbox/Snack link

https

### Steps to reproduce

2. Click on '...' 
3. Scroll down to '...' 

### gluestack-ui Version

^1.0.56

### Platform

- [ ] Expo
- [X] React Native CLI
- [ ] Next
- [ ] Web
- [X] Android
- [ ] iOS

### Additional Information

```
"@gluestack-style/react": "^1.0.56",
"@gluestack-ui/config": "^1.1.18",
"@gluestack-ui/themed": "^1.1.26",
```
I updated and I have this error:
![Screenshot_20240528-154537](https://github.com/gluestack/gluestack-ui/assets/79088170/94d01fdb-f4bd-4f46-930c-d954abf160b6)

**Comment by DaminiPandey** [comment]
@Rossella-Mascia-Neosyn Thank you for reporting this issue, we will have a look.

**Comment by DaminiPandey** [comment]
@Rossella-Mascia-Neosyn Can you share your code snippet that better demonstrates this issue?

**Comment by Rossella-Mascia-Neosyn** [comment]
The problem is that I don’t know where I have this problem

> @Rossella-Mascia-Neosyn Can you share a code snippet that better demonstrates this issue?

**Comment by dtrongphuc** [comment]
I encountered this issue in the past. It’s caused by Gluestack's icon like this:
`<Icon as={Search} size="xl" />`

**Comment by Rossella-Mascia-Neosyn** [comment]
> I encountered this issue in the past. It’s caused by Gluestack's icon like this: `<Icon as={Search} size="xl" />`

Isn’t it correct to write it like that?

**Comment by Ryochike** [comment]
@Rossella-Mascia-Neosyn 
I encountered the same issue.

I had specified `"@gluestack-ui/themed": "^1.1.22"`, but when I checked the yarn.lock file, I noticed a difference between `"@gluestack-ui/icon@0.1.20"` and `"@gluestack-ui/icon@0.1.21"`. To resolve the issue, I specified `"@gluestack-ui/themed": "1.1.22"`, and reverted back to `"@gluestack-ui/icon@0.1.20"`.

It seems that something broke with this minor update.
